aequilibrae/paths/graph.py

```diff
@@ -82,148 +86,6 @@ def default_types(self, tp: str):
         else:
             raise ValueError("It must be either a int or a float")
 
-    def create_from_geography(
```

Comment:
Do you want to add a note along the lines of "has been removed in version ..."? Not sure if it would be helpful though.

Reply:
It was used mostly on testing. It was a REALLY hard method to use, so I am not really worried about removing it. Will put in the documentation of the next release.